packages/browser-agent-core/src/features/ajax/aggregate/index.js

```diff
@@ -80,6 +79,7 @@ export class Aggregate extends FeatureBase {
 
       handle('bstXhrAgg', ['xhr', hash, params, metrics], undefined, undefined, ee)
 
+      if (this.blocked) return
```

Comment:
1. If you're trying to read _jserrors aggregate_'s `blocked`, I don't think this will happen because jserroragg.blocked isn't a property on the inherited class.
2. I think there's more code below this that isn't specifically tied to sending 'xhr' payloads to /jserror endpoint, such as the SPA stuff. If that's true, this ought to be wrapped around pertinent code.